lib/exception_hunter/hunt.rb

```diff
@@ -0,0 +1,47 @@
+module ExceptionHunter
+  class Hunt
+    def initialize(app)
+      @app = app
+    end
+
+    def call(env)
+      @app.call(env)
+    rescue Exception => e
+      catch_prey(env, e)
+      raise ex
```

Comment:
The name should match

```suggestion
    rescue Exception => exception
      catch_prey(env, exception)
      raise exception
```